.github/PULL_REQUEST_TEMPLATE

```diff
@@ -7,4 +7,7 @@ Closes #
 -
 -
 
+
+[ ] Yes I tested this before submitting it.
```

Comment:
well, better add sth like this:
a) tested in single new
b) tested in single existing
c) tested in multi

Reply:
I adjusted it.